src/main/kotlin/org/codefreak/codefreak/service/IdeService.kt

```diff
@@ -105,6 +115,39 @@ class IdeService : BaseService() {
     }
   }
 
+  fun checkIdeLiveliness(task: Task): Boolean {
```

Comment:
Naming is weird for simple boolean getter. Maybe `isIdeReady`?

Reply:
renamed it to `isIdeLive`